### DISCUSSION TITLE
Flexible space in CommandBar

### DISCUSSION THREAD

**Discussion by robkri** [bdlukaa/fluent_ui#1013] [Q&A] [answered]
I would like to create a command bar with a couple of items at its left, a couple of items in its center and a couple of items at its right, with space inbetween those three groups. The space should resize according to the overall command bar width. My current ideas are to either compute the required space myself and add CommandBarSeparators with transparaent color. Or to use a Row containing three individual command bars.

Is there a better way to do it? Like a flexible spacer that can be added inbetween CommandBarItems?

**Comment by bdlukaa** [comment, marked as answer]
I think you may want to use a RenderObject. Command bar specs doesn't describe this behavior.

**Reply by robkri** [reply]
Okay, I will try. Thank you.
